webrtc-stats.html

```diff
@@ -1529,6 +1530,26 @@ <h2>
                   6.3.2.
                 </p>
               </dd>
+              <dt>
+                <dfn>playoutDelay</dfn> of type <span class=
+                "idlMemberType">double</span>
+              </dt>
+              <dd>
+                <p>
+                  It is the sum of the time, in seconds, each <a>audio sample</a> or video frame takes from
+                  the time it is received and to the time it is played out. Given the complexities involved,
+                  the time of arrival or reception timestamp is measured as close to the network layer as 
+                  possible and the playout timestamp is measured as close to rendering as possible.
+                  In the case of audio, several samples are received in the same RTP packet, all samples
+                  will share the same reception timestamp and different playout timestamps. 
+                  Whereas, in the case of video the frame are received over several RTP packets, in this
+                  case the earliest timestamp containing the frame is counted as the reception timestamp,
+                  and the reception timestamp corresponds to when the complete frame is played out.
+                  The average playoutDelay delay can be calculated by dividing the {{playoutDelay}} with the
+                  sum of {{framesDropped}}, {{partialFramesLost}},{{framesReceived}}.
```

Comment:
This assumes that {{framesDropped}} and {{partialFramesLost}} are "played out" in some sense; you have to define a playout time for them (might be "same as following frame").

Reply:
Better to only counts framesReceived.

Reply:
Used framesDecoded